packages/react/src/components/form/Switch/Switch.mdx

```diff
@@ -1,6 +1,6 @@
 import { Meta, Canvas, Controls, Primary } from '@storybook/blocks';
 import { CssVariables } from '@doc-components';
-import css from '@digdir/designsystemet-css/switch.css?inline';
+import css from '@digdir/designsystemet-css/input.css?inline';
```

Comment:
Since this is a part of our input component, maybe we should omit the css vars on this docs page?

Reply:
Yeah, though of that too. Just added this since it is the same for radio and checkbox, and we can evaluate what is best when we are doing documentation? ☺️